build_tools/cmake/build_e2e_test_artifacts.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Copy and modify `build_tools/cmake/build_benchmarks.sh`. Eventually `build_tools/cmake/build_benchmarks.sh` will be removed.